ambry-cloud/src/main/java/com.github.ambry.cloud/CloudMessageReadSet.java

```diff
@@ -52,6 +52,9 @@ public long writeTo(int index, WritableByteChannel channel, long relativeOffset,
       if (blobReadInfo.isPrefetched()) {
         ByteBuffer outputBuffer = blobReadInfo.getPrefetchedBuffer();
         outputBuffer.flip();
```

Comment:
Do we still need this flip()?

Reply:
No we dont. Good catch. Will remove them.